group_user_management/models/res_groups.py

```diff
@@ -0,0 +1,70 @@
+#  Copyright (c) 2024- Le Filament (https://le-filament.com)
+#  Copyright (c) 2004- Odoo S.A.
+#  License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from lxml import etree
+from lxml.builder import E
+
+from odoo import api, models
+
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+
+
+class GroupsView(models.Model):
```

Comment:
```suggestion
class ResGroups(models.Model):
```

Reply:
Thanks @legalsylvain  for looking into it.
I kept it as is since Odoo names the class that way here : https://github.com/odoo/odoo/blob/66e7c919c57d5495aaf7705b641bed2070e1adf8/odoo/addons/base/models/res_users.py#L1367 (which is the same file where they already have Groups class defined) and also here : https://github.com/odoo/odoo/blob/66e7c919c57d5495aaf7705b641bed2070e1adf8/addons/account/models/res_users.py#L30
But fine with me I can rename that ResGroups

Reply:
Yes. Odoo doesnt have strong convention on tha topic. OCA does. 
Better to follow OCA rules